src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,50 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum LottoResult {
+    NO_REWARD(0, false, 0, "상금이 없습니다."),
+    THREE_MATCHING(3, false, 5000, "3개 일치 (5,000원) - "),
+    FOUR_MATCHING(4, false, 50000, "4개 일치 (50,000원) - "),
+    FIVE_MATCHING(5, false, 1500000, "5개 일치 (1,500,000원) - "),
+    FIVE_MATCHING_BONUS(5, true, 30000000, "5개 일치, 보너스 볼 일치 (30,000,000) - "),
+    SIX_MATCHING(6, false, 2000000000, "6개 일치 (2,000,000,000원) - ");
+
+    private final int count;
+    private final boolean includeBonus;
+    private final int reward;
+    private final String description;
+
+    LottoResult(int count, boolean includeBonus, int reward, String description) {
+        this.count = count;
+        this.includeBonus = includeBonus;
+        this.reward = reward;
+        this.description = description;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public int getReward() {
+        return reward;
+    }
+
+    public static LottoResult of(int count) {
```

Comment:
```suggestion
    public static LottoResult from (int count) {
```
하나의 매개 변수를 처리하면 이렇게 되는 것이 적절한 표현인 것 같습니다.